.github/workflows/Test.yml

```diff
@@ -16,671 +16,692 @@ jobs:
 #
 
   doc:
-    runs-on: ubuntu-latest
     name: '📓 Docs'
-    steps:
+    runs-on: ubuntu-latest
 
-    - name: '🧰 Checkout'
-      uses: actions/checkout@v2
-
-    - name: '⛴️ Build ghdl/doc'
-      run: |
-        docker build -t ghdl/doc . -f- <<-EOF
-        FROM ghdl/debug:base
-        ENV PYTHONPATH=/opt/ghdl/pyGHDL
-        COPY . /opt/ghdl
-        RUN cd /opt/ghdl && ./configure && make && make install
-        EOF
-
-    - name: '📚 Run gnatdoc'
-      run: |
-        cat > run.sh <<-EOF
-        #!/usr/bin/env sh
-        gnatdoc -P./ghdl
-        mkdir /src/public
-        mv gnatdoc /src/public/gnatdoc
-        EOF
-        chmod +x run.sh
-        docker run --rm -v $(pwd):/src -w /opt/ghdl ghdl/doc /src/run.sh
-
-    - name: '📓 BuildTheDocs (BTD)'
-      if: github.event_name != 'pull_request'
-      uses: buildthedocs/btd@v0
-      with:
-        token: ${{ github.token }}
-
-    #- run: nroff -man doc/_build/man/ghdl.1
-
-    - name: '📤 Upload artifact: HTML and LaTeX'
-      if: github.event_name != 'pull_request'
-      uses: actions/upload-artifact@v2
-      with:
-        name: doc
-        path: |
-          doc/_build/html
-          doc/_build/latex
-          doc/_build/man
-
-    - name: '📤 Upload artifact: ghdl.1'
-      if: github.event_name != 'pull_request'
-      uses: actions/upload-artifact@v2
-      with:
-        name: man
-        path: doc/_build/man/ghdl.1
+    steps:
```

Comment:
Please, either remove the space before, or leave a blank line both before and after `steps`.